__tests__/app.test.js

```diff
@@ -245,6 +245,60 @@ describe('POST /api/articles/:article_id/comments', () => {
      })
     })
 
+describe('PATCH /api/articles/:article_id', () => {
```

Comment:
we could have another test for when inc_votes is missing, and one where it ignores other unnecessary properties